OrangeHouse.qml

```diff
@@ -1,14 +1,27 @@
 import Qt3D.Core 2.0
 import Qt3D.Render 2.0
 import Qt3D.Extras 2.0
+import QtQuick 2.7
 
 Entity {
-	components: [
-		SceneLoader {
-			source: "/models/orangehouse.qgltf"
-		},
-		Transform {
-			translation: Qt.vector3d(0, 0, -0.038) // the marker is on the top of the house
-		}
-	]
+    property matrix4x4 t
+
+    components: [
+        Transform {
+            matrix: t.inverted()
+        }
+    ]
+
+    Entity {
+
+        components: [
+            SceneLoader {
+                source: "/models/orangehouse.qgltf"
+            },
+            Transform {
+                rotation: fromAxisAndAngle(Qt.vector3d(0,0,1), 180) // buggy 3d model?
```

Comment:
Initially that was not necessary, so I am not sure the 3D model is buggy, maybe the world to camera transform has a problem?